deploy/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: nginx-gateway
```

Comment:
shall we use `nginx-kubernetes-gateway` ? this will be consistent with the name of container image too.